source/command-monitoring/tests/command.json

```diff
@@ -7,6 +7,7 @@
   ],
   "collection_name": "test",
   "database_name": "command-monitoring-tests",
+  "read_preference": "primary",
```

Comment:
This seems redundant to the modifiers below.  Let's either specify once at the top of the test spec, or in each operation, but not both